src/components/NavHeader/NavHeader.spec.tsx

```diff
@@ -0,0 +1,24 @@
+/** @jest-environment jsdom */
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import { NavHeader } from "./NavHeader";
+import { HEADER_ITEMS } from "./constants";
+import { ThemeProvider } from "styled-components";
+import Theme from "../../../style/Theme";
+
+describe("Components / <NavHeader />", () => {
+
+  it("render component", async () => {
+    render(
+      <ThemeProvider theme={Theme}>
+        <NavHeader />
+      </ThemeProvider>
+    );
+
+    expect(screen.getByText(HEADER_ITEMS[0])).toBeInTheDocument();
+    expect(screen.getByText(HEADER_ITEMS[1])).toBeInTheDocument();
+    expect(screen.getByText(HEADER_ITEMS[2])).toBeInTheDocument();
+    expect(screen.getByText(HEADER_ITEMS[3])).toBeInTheDocument();
+    expect(screen.getByText(HEADER_ITEMS[4])).toBeInTheDocument();
```

Comment:
```suggestion
    HEADER_ITEMS.forEach((item) => expect(screen.getByText(item)).toBeInTheDocument());
```
if your array changes the size, the test will always keep working :D